src/DotNet.Status.Web/Controllers/GitHubHookController.cs

```diff
@@ -49,16 +51,26 @@ private async Task EnsureLabelsAsync()
         }
 
         [GitHubWebHook(EventName = "issues")]
-        public async Task<IActionResult> IssuesHook(IssuesHookData data)
+        public async Task<IActionResult> IssuesHook(JsonElement data)
```

Comment:
Couldn't we take `string` here instead of `JsonElement`? In current implementation we deserialize data from input to `JsonElement` just to call `ToString()` on it to convert it back to JSON and deserialize again. This does not seem too optimal.

Reply:
We can't. Because the http headers of requests claims body is json having it string would expect body to be in format `"string-value"` but it is `{ ... }`